tests/unit_tests/pyanaconda_tests/modules/storage/test_module_device_tree.py

```diff
@@ -849,6 +849,24 @@ def test_set_device_mount_options(self):
         self.interface.SetDeviceMountOptions("dev1", "")
         assert dev1.format.options == "defaults"
 
+    def test_get_required_mount_points(self):
+        """Test GetRequiredMountPoints."""
+        result = self.interface.GetRequiredMountPoints()
+        assert isinstance(result, list)
+        assert len(result) != 0
+
+        for mp in result:
+            assert mp["value"] is not None
+            assert mp["fstype"] is not None
+
+        # we are always adding / so it's a good candidate for testing
+        root = next(r for r in result if r["value"] == get_variant(Str, "/"))
+        assert root is not None
+        assert root["value"] == get_variant(Str, "/")
```

Comment:
Please, consider to use `get_native(root)` to check the content of the structure. The current test will not catch newly added fields in the future.